### PR TITLE
move xeh declarations to description.ext

### PR DESCRIPTION
**Pull request by bux**
- Change: Move XEH declarations to description.ext (@bux).

**When merged this pull request will:**
- This change will allow mission makers to use the XEH's too because you can't have duplicate class declarations.
- same change as with the CfgFunctions PR I did some time ago #465 

**Final test:**
- [x] local
- [x] server

### CODE REVIEW

File: =BTC=co@30_Hearts_and_Minds.Altis/description.ext [hunk 1]
```diff
@@ -33,5 +33,22 @@ class CfgFunctions {
...
+    };
+    class Air {
+        #include "core\fnc\eh\xeh_InitPost_EH_Vehicle.hpp"
+    };
+    class Ship {
+        #include "core\fnc\eh\xeh_InitPost_EH_Vehicle.hpp"
+    };
+    class CAManBase {
+        //#include "core\fnc\eh\xeh_InitPost_EH_CAManBase.hpp"
+	};
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/description.ext]: Just change the little tab to 4 spaces for consistency.

Comment (bux) [reply =BTC=co@30_Hearts_and_Minds.Altis/description.ext]: done

**Comment by Vdauphin** [comment]
Hello,
Why did you close it ?
Cheers

**Comment by bux** [comment]
eh ... i didn't ... not on purpose ...

**Pull request merged**